### ISSUE TITLE
Can't delete multiple channels at once

### ISSUE THREAD

**Issue by canthefason** [slact/nchan#280]
I am able to publish messages to multiple messages with this config:

```
location ~ /multipub-split/(.*)$ {
    nchan_publisher;
    nchan_channel_id "$1";
    nchan_channel_id_split_delimiter ",";
}
```

But unfortunately, it is not possible to delete multiple channels with this config. So when this is the case, I am sending separate delete requests for each channel. 

Is it something intentional? Depending on the documentation, it does not say that Nchan has support to delete multiple channels, so I assume that it is not a bug actually. 

I have currently tested this on Nchan version 1.0.3

**Comment by slact** [comment]
This behavior is not well-defined in the documentation.

A DELETE to a multiplexed channel (channel with several ids) only deletes the multiplexing channel and not the associated individual channels. That means the DELETE kicks out all subscribers to the multiplexed channel, but does not delete the messages in any of the individual channel, nor does it kick their subscribers.

This behavior is inconsistent with how POSTs are handled. A DELETE to a multiplexed channel should delete everything -- the multiplexing channel, and the underlying individual channels.

I'll fix this and document it better.

**Comment by slact** [comment]
fixed in a95fbd25d063364c7472837f4462ec9027cc97cf , updated documentation in eeb9793c91ce2752dec7067d6f977ba97127b49e . Let me know if this behaves the way you expect.

**Comment by canthefason** [comment]
👍 thanks for this update!
